docs/get-started/3d-mesh/index.md

```diff
@@ -0,0 +1,303 @@
+
+Now let's see whether we made an improvement or not. We will use the training data as index and the testing data as query. We'll embed them both
+and then search for test 3d objects (the query) in our training data (index). We do this using the pretrained model and the finetuned model and then compare top matches.
+
+We'll use two metrics to evaluate our two models (pretrained, straight out of the box model and finetuned model) on the search task:
+
+**mAP@k** : We'll calculate the average precision at 1, 5 and 10, then we'll calculate the mean of those average precisions for all documents in our test data (these are our queries) because we care about how accurate and precise our retrieved 3D objects are.
+
+**mNDCG@k**: We'll calculate NDCG at 1,5 and 10, then we'll calculate the mean of those for all documents in our test data  because we care about the order of the retrieved 3D objects.
+
+````{dropdown} Complete source code
+
+```python
+from finetuner.tuner.evaluation import Evaluator
+
+train = DocumentArray.load_binary('../train_data_modelnet.bin') # load train dataset
+val = DocumentArray.load_binary('../test_data_modelnet.bin') # load test dataset
+
+train.apply(preprocess)
+val.apply(preprocess)
+
+tuned_model = MeshDataModel(model_name='pointconv', embed_dim=512) # create pointconv model with 512 dimensions
+pretrained_model = MeshDataModel(model_name='pointconv', embed_dim=512)
+
+tuned_model.load_state_dict(torch.load('checkpoints/finetuned-pointconv-d512.pth')) # load finetuned weights
+
+tuned_model.eval()
+pretrained_model.eval()
+
+evaluator_finetuned = Evaluator(val, train, tuned_model)
+print(evaluator_finetuned.evaluate())
+
+evaluator_pretrained = Evaluator(val, train, pretrained_model)
+print(evaluator_pretrained.evaluate())
+```
+````
+
+The difference is shown in the tables below:
+
+| mAP@k  | pre-trained | fine-tuned |
+|--------|-------------|------------|
+| mAP@1  | 0.147       | 0.719      |
+| mAP@5  | 0.113       | 0.697      |
+| mAP@10 | 0.100       | 0.686      |
+
+| mNDCG@k  | pre-trained | fine-tuned |
+|--------|-------------|--------------|
+| mNDCG@1  | 0.563       | 0.927      |
+| mNDCG@5  | 0.617       | 0.931      |
+| mNDCG@10 | 0.647       | 0.935      |
+
+Now let's do some queries ourselves and check the visualizations
+
+````{dropdown} Complete source code
+
+```python
+train_da = DocumentArray.load_binary('../train_data_modelnet.bin') # load train dataset
+eval_da = DocumentArray.load_binary('../test_data_modelnet.bin') # load eval dataset
+
+train_da.apply(preprocess)
+eval_da.apply(preprocess)
+
+tuned_model = MeshDataModel(model_name='pointconv', embed_dim=512) # create pointconv model with 512 dimensions
+tuned_model.load_state_dict(torch.load('checkpoints/finetuned-pointconv-d512.pth'))
+tuned_model.eval()
+
+train_da.embed(tuned_model, batch_size=128, device='cuda')
+eval_da.embed(tuned_model, batch_size=128, device='cuda')
+
+eval_da.match(train_da, limit=10)
+```
+````
+
+Query:
```

Comment:
Can you check if it is possible to create a table here, so that query, pretrained and finetuned results can be shown side to side? This would improve the presentation I think

Reply:
I tried different solutions including html, I can't seem to get it done

Reply:
I don't know if you have tried this already but it may help: https://stackoverflow.com/a/24320279
I was able to show side by side in my PyCharm 
![image](https://user-images.githubusercontent.com/6537525/151143901-da335e8c-66cf-4c8b-9fe1-8800b7a00b73.png)

Reply:
are they still gifs ? I tried it and it didn't work, can you perhaps share code

Reply:
Yes they are still rendered as gif. Code is taken from the above link mentioned, I am attaching the snapshot.

![Screenshot 2022-01-26 at 13 19 56](https://user-images.githubusercontent.com/6537525/151161932-aea24f64-b1fc-46aa-85bc-302ab363d15b.png)

https://user-images.githubusercontent.com/6537525/151161605-50c54949-5007-4f31-85e6-36ee510aadb5.mov

Reply:
this is what I'm getting locally
![Screenshot from 2022-01-26 15-05-22](https://user-images.githubusercontent.com/40893766/151177530-6e0630d7-cc1c-40d9-a920-6a2349416a3b.png)
 
![Screenshot from 2022-01-26 15-06-43](https://user-images.githubusercontent.com/40893766/151177598-3cb03622-96ce-4b60-a336-d8da5570ced6.png)